internal/support/notifications/config_system.go

```diff
@@ -0,0 +1,24 @@
+package notifications
```

Comment:
Don't use a separate file for this. Just put the function at the bottom of router.go. I would tell you to call it config.go but when I come through for ConfigV2, I will create a config.go that contains the ConfigurationStruct for the service to be consistent with other services.